### DISCUSSION TITLE
Is there any helper for joins?

### DISCUSSION THREAD

**Discussion by hulcon** [tekartik/sqflite#785] [Q&A] [answered]
Is there any helper method/function for joins (selecting data from more than one table) or do we have to use raw query?

**Comment by alextekartik** [comment, marked as answer]
It does not include helpers for complex queries. Personally I create views (using raw SQL) during onCreate. Some views are joins. Then I can use the simple query API (using the view as the table argument)

**Reply by hulcon** [reply]
Thanks. Can you please provide a code snippet/example to clear the concept?

**Comment by alextekartik** [comment]
I don't have specific easy examples to share. Basically you use `CREATE VIEW`: https://www.w3schools.com/sql/sql_view.asp
I guess you can find better CREATE VIEW SQLite example online: https://www.w3resource.com/sql/creating-views/create-view-with-join.php

**Reply by hulcon** [reply]
Thanks a lot!
